service/controller/key/key.go

```diff
@@ -346,11 +341,6 @@ func MastersSubnetCIDR(customObject providerv1alpha1.AzureConfig) string {
 	return customObject.Spec.Azure.VirtualNetwork.MasterSubnetCIDR
 }
 
-// WorkerCount returns the desired number of workers.
```

Comment:
Not sure why this was removed. It broke `instance` handler, so I'll revert it for now.

Reply:
These might have been cleanup work towards dropped in-built workers handling?